### DISCUSSION TITLE
go2rtc update

### DISCUSSION THREAD

**Discussion by cryd-s** [blakeblackshear/frigate#7754] [Ask A Question]
Hi,
is there a easy way to update go2rtc from 1.2 to the newest release?
Sorry but its first time using docker

**Comment by NickM-27** [comment]
https://docs.frigate.video/configuration/advanced#custom-go2rtc-version

**Reply by cryd-s** [reply]
sorry for asking noob questions but do u mean the config in the docker or the config folder in the lxc. 

how to rename the build?

dont know how to give permissons. 

Is there a tutorial in yt u can recommend?

**Reply by NickM-27** [reply]
The folder that is mapped to /config in the frigate container.

To give permissions you can just run `chmod +X go2rtc` after the file has been downloaded

**Comment by cybertza** [comment]
I think for 107 to work there would have to be a separate config file, some it writes key pairs to the config, but I believe Frig will recreate the config from the settings in its config file, I am referring to HomeKit for this in specific.
